Frontend-v1-Original/components/options/stake.tsx

```diff
@@ -227,35 +224,40 @@ export function Stake() {
           <div>Total staked</div>
           <div>${formatCurrency(totalStakedValue)}</div>
         </div>
-        <div className="flex items-center justify-between">
-          <div className="flex items-center justify-start gap-2">
-            <div>APR</div>
-            {displayedRewardTokens && displayedRewardTokens.length > 0 && (
-              <div className="flex items-center justify-center gap-1">
-                {displayedRewardTokens.map((token) => (
+        <div className="my-1 flex items-center justify-between">
+          <div>APR</div>
+          <div className="flex flex-col">
+            {displayedRewardTokens && displayedRewardTokens.length > 0 ? (
+              displayedRewardTokens.map((token) => (
+                <div
+                  key={token.address}
+                  className="flex items-center justify-end gap-2"
+                >
+                  <div>
+                    {"aprRange" in token
+                      ? `${formatCurrency(
+                          token.aprRange[0]
+                        )} % - ${formatCurrency(token.aprRange[1])} %`
+                      : `
+                  ${formatCurrency(token.apr)} %
+                  `}
+                  </div>
                   <img
-                    key={token.address}
                     src={token.logoUrl ?? "/tokens/unknown-logo.png"}
                     alt={`${token.symbol} logo`}
-                    className="h-5 w-5 rounded-full"
+                    className="block h-5 w-5 rounded-full"
```

Comment:
```suggestion
                    className="block h-5 w-5 rounded-full"
                    title={token.symbol}
```
We use same icon for both flow and oFlow, it could be confusing, so maybe add a title to help?